src/components/TimeEstimator/__tests__/TimeEstimator.spec.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @file Time estimator tests.
+ * @copyright IBM Security 2019
+ */
+
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { Default as TimeEstimator } from '../TimeEstimator.stories';
```

Comment:
This is pretty cool, but I wonder how well it would work when a component that has a bunch of knobs?

Reply:
True - I'd be interested to see how well it integrates with event handlers, but the real value I see is that we now have synchronization between what we display in Storybook and what we unit test.